README.md

```diff
@@ -30,3 +30,13 @@ jenkins-jobs --conf /path/to/your/jenkins_jobs.ini test /path/to/your/template.y
 ```
 jenkins-jobs --conf /path/to/your/jenkins_jobs.ini update /path/to/your/template.yaml
 ```
+
+### Generate inline script jobs
+```
+./scripts/generate_inline_script_pipeline_job -j /path/to/your/template.yaml -o /path/to/your/generated/jobs
+```
+
+### Configure all jobs and views
+```
+./scripts/configure_jenkins /path/to/your/jenkins_jobs.ini
```

Comment:
This didn't work for me when `jenkins-job.ini` was not located in `ci-cd` directory or its subdirectories. Using both relative and absolute path to e.g. `ci-cd/jenkins_jobs.ini` worked as expected. Not a blocker for a merge though.

Reply:
Hadn't even tried that tbh, but yeah the script is assuming you are working with a file in this directory or a sub directory of it as you have pointed out.